src/components/Map/SearchPane/Drawer/DrawerContent/BusinessTypeEntities/BusinessTypeEntities.tsx

```diff
@@ -1,27 +1,32 @@
-import React, { useEffect } from 'react';
-import { Link } from 'react-router-dom';
-import { IonItem, IonText } from '@ionic/react';
+import React from "react";
+import { IonItem, IonText } from "@ionic/react";
+import { useNavigateBottomDrawer } from "../../drawerUtils";
```

Comment:
Let's stick with absolute paths rather than relative for imports. I did wonder about `drawerUtils` vs `DrawerUtils` but I see it's not a Component, so I think you've made the right choice.